src/main/drivers/sensor.h

```diff
@@ -34,10 +34,13 @@ typedef enum {
     CW270_DEG_FLIP = 8
 } sensor_align_e;
 
-typedef union busDevice_t {
+typedef union busDevice_u {
```

Comment:
Ofcourse. Just generating ideas here. I think we need to have run-time bus selection. I.e. for `PIXRACER` target we have HMC5883L compass on SPI and we might have it on external I2C. Some setups might get away with onboard compass sensor - for those it would be good to have an option to chose bus at run-time.

Similarly for `REVO` target.

Also, we might want to have an option to add external IMU (gyro+accel) on external bus (SPI or I2C) - as a secondary gyro.